node/core/dispute/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+mod votesdb;
+mod metrics;
+
+pub use votesdb::*;
```

Comment:
Typically we have been giving each subsystem their own crate. So `votesdb` would be separate from `dispute_participation`. Any reason why not?

Reply:
The were both in one issue and both seemed to be fairly small in scope and only are relevant to disputes, hence I started with one subsystem.